db/schemas.py

```diff
@@ -71,10 +75,65 @@ def get_all_schemas(engine):
     return inspector.get_schema_names()
 
 
+def get_related_tables(engine, schema):
+    metadata = MetaData()
+    pg_tables = Table("pg_tables", metadata, autoload_with=engine)
+    sel = (
+        select(pg_tables.c.schemaname, pg_tables.c.tablename).
+        where(pg_tables.c.schemaname == schema)
+    )
+    with engine.begin() as conn:
+        results = conn.execute(sel).fetchall()
+
+    related_tables = [tables.reflect_table(table, schema, engine)
+                      for schema, table in results]
```

Comment:
You can get these related tables by initializing the metadata with args:
```python
metadata = Metadata(bind=engine, schema=schema)
```
Then, use:
```python
metadata.reflect()
```
Then, you can get all the tables in that schema via
```python
metadata.tables
```